amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/index.tsx

```diff
@@ -83,12 +102,22 @@ class DetailListItem extends React.Component<DetailListItemProps, DetailListItem
             <img className={'icon ' + (this.state.isExpanded ? 'icon-up' : 'icon-down')}/>
           }
         </div>
-        <div className={'body-secondary-3 description ' + (isExpandable && !this.state.isExpanded ? 'truncated' : '')}>
-          <ColumnDescEditableText
-            columnIndex={this.props.index}
-            editable={metadata.is_editable}
-            value={metadata.description}
-          />
+        <div className='desc-container'>
```

Comment:
nit: I have a preference for using the full term instead of abbreviations. 

I think about how abbreviations are a slippery slope towards a state where we have different names that reference similar things littered in the code (e.g: `desc`, `dsc`, `dscrptn`, +more) and also that extensive adhoc abbreviations risk becoming unintuitive for everyone -- but especially for non-native English speakers.